src/rt/array/concat.d

```diff
@@ -0,0 +1,83 @@
+module rt.array.concat;
+
+private extern (C) void[] _d_arraycatnTX(const TypeInfo ti, byte[][] arrs);
+
+T[] _d_arraycatnTX(T)(scope T[][] arrs) @trusted
```

Comment:
Please add DDoc comment including `Params` and `Returns`.  A note about `@trusted` being a temporary hack would also be welcome.